api/EffectTiming.json

```diff
@@ -253,10 +253,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": false
+              "version_added": "13.1"
```

Comment:
`easing` has a subfeature "jump- keywords for steps()" which is left false. Should it be?

Reply:
I've updated it by copying the data from the same feature within the CSS data.